### ISSUE TITLE
7 Money Lessons

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#55]
# TL;DR
Mindset to achieve financial stability, are little things and every day thing

# Key Takeaways

- **It’s not about how much you make.** It’s about how much you save.

- **Never underestimate Cash On Hand.** Part of saving money means having a balance between cash you invest, and cash you keep “on hand.” A good rule of thumb a mentor of mine gave me was to always have at least 3 months of living expenses readily available.

- **Savings** Save 50% after tax.

- **Have a plan for the worse-case scenario.** Also saving money is to be able to navigate another recession

- **Work to create multiple streams of revenue.** Your goal should be to create ~7 streams of revenue for yourself, so that if one declines, the others can keep the ship moving in the meantime (while you either repair what went wrong, or create a new stream of revenue for yourself).

- **Spend time around financially prudent people.** Instead, I encourage you to seek out people who are very responsible with money. For me, I sought out people I knew were successful and yet didn’t live overly lavish lifestyles: family friends, people in my network, etc. I wanted to know how they treated their money, so I could learn how to do the same.

- **Don’t ever “bet it all.”** But the reason I strong discourage this way of operating is because it the beginning of an even worse habit that can linger over time. You don’t want to get used to “betting it all.” You want to get in the habit of betting what you’re willing to lose, without compromising your long-term financial position.

# Link
https://medium.com/the-ascent/7-hard-money-lessons-people-are-going-to-learn-during-this-pandemic-9936de65aa65
